website/docs/docs/building-a-dbt-project/building-models/using-variables.md

```diff
@@ -20,7 +20,7 @@ Variables can be defined in two ways:
 :::info New in v0.17.0
 
 The syntax for specifying vars in the `dbt_project.yml` file has changed in
-dbt v0.17.0. See the [migration guide](migration-guide/upgrading-to-0-17-0)
```

Comment:
Should the original links still work? Do we want redirects on them, as well, in case those links are still floating around out there in the world or in search?